common/handy/lists.go

```diff
@@ -1,26 +1,62 @@
 package handy
 
-// Lists is a collection of named lists.
-type Lists[T any] map[string][]T
+// Lists is a dynamic list of identifiable slices.
+// Each slice is associated to the unique comparable object.
+type Lists[ID comparable, V any] map[ID][]V
```

Comment:
A list would generally mean an array in usual nomenclature, so I would call this something like IndexedLists, MapOfLists (very verbose I know), or something else

Reply:
`IndexedLists` is a good choice.